SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -61,18 +61,14 @@ class FLocalDeploymentManager
 
 	bool bLocalDeploymentRunning;
 	bool bSpatialServiceRunning;
+	bool bSpatialServiceInProjectDirectory;
```

Comment:
It would be safer if bSpatialServiceInProjectDirectory is initialised in constructor.

Reply:
Done